core/src/main/java/org/apache/gravitino/listener/api/info/Either.java

```diff
@@ -40,6 +41,7 @@ public final class Either<L, R> {
    * @param <R> Right type
    */
   public static <L, R> Either<L, R> left(L value) {
+    Preconditions.checkNotNull(value, "Left value cannot be null");
```

Comment:
Can we use checkArgument instead of checkNotNull, checkNotNull will throw an NPE, I think we should not throw NPE delibrately.